tests/trigonometry.cpp

```diff
@@ -31,6 +35,19 @@ TEST(trigonometry, cos)
         auto cos_fixed = static_cast<double>(cos(P(flt_angle)));
         EXPECT_TRUE(HasMaximumError(cos_fixed, cos_real, MAX_ERROR_PERC));
     }
+
+    // Boundary-value analysis
+    constexpr int32_t raw_values[] = {INT32_MIN, 2147380703, 2147380704, INT32_MAX};
+    for (int32_t i=0; i<size(raw_values); i++)
```

Comment:
Can you use a ranged for loop here? Makes for a simpler for loop and removes the need for the `size` method.
(same thing in the other test).

Reply:
Done. 
(I'm currently working on an old mixed C / C++ project... so I often end up with such constructs unintentionally... 🙂  )